python/examples/datasets/weather.ipynb

```diff
@@ -0,0 +1,1553 @@
+            "        - temperature at 2m, K\n",
+            "        - Input\n",
+            "        - all\n",
+            "    *   - wrf_wind_u\n",
+            "        - wind U component, m/s\n",
+            "        - Input\n",
+            "        - all\n",
+            "    *   - wrf_wind_v\n",
+            "        - wind V component, m/s\n",
+            "        - Input\n",
+            "        - all\n",
+            "    *   - wrf_wind_v\n",
+            "        - wind V component, m/s\n",
+            "        - Input\n",
+            "        - all\n",
+            "    *   - rain_rate\n",
+            "        - avg rain rate between two horizons, mm/h\n",
+            "        - Input\n",
+            "        - all\n",
+            "    *   - snow_rate\n",
+            "        - avg snow rate between two horizons, mm/h\n",
+            "        - Input\n",
+            "        - all\n",
+            "    *   - hail_velocity\n",
+            "        - hail velocity on two horizons, mm/h\n",
+            "        - Input\n",
+            "        - all\n",
+            "    *   - wrf_temperature_grad\n",
+            "        - difference between temperatures at 2m on adjacent horizons, K\n",
+            "        - Input\n",
+            "        - all\n",
+            "    *   - rain_accumulated_grad\n",
+            "        - rain accumulated from cmc gentime difference between adjacent horizons, mm\n",
+            "        - Input\n",
+            "        - all\n",
+            "    *   - snow_accumulated_grad\n",
+            "        - snow accumulated from cmc gentime difference between adjacent horizons, mm\n",
+            "        - Input\n",
+            "        - all\n",
+            "    *   - ice_rain_grad\n",
+            "        - ice rain accumulated from cmc gentime difference between adjacent horizons, mm\n",
+            "        - Input\n",
+            "        - all\n",
+            "    *   - iced_graupel_grad\n",
+            "        - iced graupel accumulated from cmc gentime difference between adjacent horizons, mm\n",
+            "        - Input\n",
+            "        - all\n",
+            "    *   - cloud_coverage_grad\n",
+            "        - difference between low level cloud coverage on adjacent horizons, %\n",
+            "        - Input\n",
+            "        - all\n",
+            "\n",
+            "References\n",
+            "----------\n",
+            "\n",
+            "- Malinin, Andrey, et al. \"Shifts: A dataset of real distributional shift across multiple large-scale tasks.\" arXiv preprint arXiv:2107.07455 (2021).\n",
+            "\n"
+          ]
+        }
+      ],
+      "source": [
+        "print(Weather.describe())"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "note: the output was cleared as `describe()` will print a rather lengthy description."
```

Comment:
so what is the correct approach to use this method? where should users refer to its description?

Reply:
This is the correct method. I cleared the output because it would pollute the notebook, but the user should use this method if he wants to get the full description of the dataset.

Is the note confusing? Maybe I should remove it?

Reply:
I truncated the message in the last commit.